apisix/plugins/limit-count.lua

```diff
@@ -248,10 +250,11 @@ function _M.access(conf, ctx)
         key = ctx.var["remote_addr"]
     end
 
+    -- here we add a separator ':' to mark the boundary of the prefix and the key itself
     if not conf.group then
-        key = key .. ctx.conf_type .. ctx.conf_version
+        key = ctx.conf_type .. ctx.conf_version .. ':' .. key
     else
-        key = key .. conf.group
+        key = conf.group .. ':' .. key
```

Comment:
I wonder if we should add a note here to alert those who use the key to monitoring or something else.

Reply:
Could you give an example for the note?

Reply:
How about:
```bash
-- key format changed from `key*` to `*key` for constant key type support
```

Reply:
Err...
I don't want to comment on all the changes in the code, unless it is something that affects public API. AFAIK, we don't describe the key format in the doc. So I think such change is just mplementation details.

Reply:
Make Sense, leave the comment here should do the trick as well.